sass/components/_liftup-grid.scss

```diff
@@ -35,7 +35,7 @@
   }
 
   &.theme-small {
-    background-color: $black !important;
+    background-color: $pitchblack !important;
```

Comment:
No idea why this should have ```!important```, but decided not to touch it to not break anything.